packages/components/pill/stories/Pill.stories.tsx

```diff
@@ -51,7 +62,13 @@ export const CustomHandleComponent = (args: PillProps) => (
   />
 );
 
-export const PillVariants = (args: PillProps) => (
+customHandleComponent.args = {
+  label: 'example.user@contentful.com',
+  onClose: action('clicked'),
+  onDrag: action('dragged'),
+};
+
+export const PillVariants: Story<PillProps> = (args: PillProps) => (
```

Comment:
A small thing that came to my mind is that maybe we could extend the existing overview story to contain the `deleted` variant with a close icon.